src/app/store/states/formbox-state.ts

```diff
@@ -16,5 +18,15 @@ export interface FormBoxState {
  */
 export const INITIAL_STATE: FormBoxState = {
   absenderliste: { selected: undefined, pal: [] },
-  template: { status: LoadingStatus.None, overrideFrags: [], documentCommands: [] } as TemplateState
+  template: { status: LoadingStatus.None, fragments: [], overrideFrags: [], documentCommands: [] } as TemplateState,
```

Comment:
Den Initial State für expressionEditor in der Datei src/app/store/states/expression-editor-state.ts definieren und dann per import hier verwenden.